scroll-animations-1/Overview.bs

```diff
@@ -347,6 +347,57 @@ spec: cssom-view-1; type: dfn;
 	In the Web Animations API,
 	they can be represented anonymously by a {{ViewTimeline}} object.
 
+## View Progress Timeline Ranges ## {#view-timelines-ranges}
+
+	[=View progress timelines=] define the following [=named timeline ranges=]:
+
+	<dl dfn-for="animation-timeline-range" dfn-type="value">
+		<dt><dfn>cover</dfn>
+		<dd>
+			Represents the full range of the [=view progress timeline=]:
+			* 0% progress represents the position at which
+				the [=end=] [=border edge=] of the element’s [=principal box=]
+				coincides with the [=start=] edge of its [=view progress visibility range=].
+			* 100% progress represents the position at which
+				the [=start=] [=border edge=] of the element’s [=principal box=]
+				coincides with the [=end=] edge of its [=view progress visibility range=].
+
+		<dt><dfn>contain</dfn>
+		<dd>
+			Represents the range during which the [=principal box=]
+			is either fully contained by, or fully covers,
+			its [=view progress visibility range=] within the [=scrollport=].
+			* 0% progress represents the earlier position at which:
+				* the [=start=] [=border edge=] of the element’s [=principal box=]
+					coincides with the [=start=] edge of its [=view progress visibility range=].
+				* the [=end=] [=border edge=] of the element’s [=principal box=]
+					coincides with the [=end=] edge of its [=view progress visibility range=].
+			* 100% progress represents the later position at which:
+				* the [=start=] [=border edge=] of the element’s [=principal box=]
+					coincides with the [=start=] edge of its [=view progress visibility range=].
+				* the [=end=] [=border edge=] of the element’s [=principal box=]
+					coincides with the [=end=] edge of its [=view progress visibility range=].
+
+		<dt><dfn>entry</dfn>
```

Comment:
This should be `enter`

Reply:
Should it? Kinda feels like phase names make more sense as nouns than verbs, but I'm open to debating it.